src/Header/Header.tsx

```diff
@@ -53,7 +53,11 @@ const Header = () => {
 
 const TodayButton = ({ handleClick }: { handleClick: () => void }) => {
   return (
-    <button className={styles.headerButton} onClick={handleClick}>
+    <button
+      className={styles.headerButton}
+      onClick={handleClick}
+      aria-label="todayButton"
```

Comment:
Why did you add these? These are meant for screen readers.
```suggestion
      aria-label="todayButton"
```

Reply:
well I have to query it somehow, this button has no text, so I googled a solution to use aria-label. Could be data-testid but as I understand they are not preferable. I use it here: `userEvent.click(screen.getByRole('button', { name: /arrowRight/i }));`

Reply:
Just use label `userEvent.click(screen.getByRole('button', { name: /Today/i }));`

and remove this one ;)